Packs/MicrosoftGraphApplications/Integrations/MicrosoftGraphApplications/MicrosoftGraphApplications_test.py

```diff
@@ -137,3 +138,276 @@ def test_service_principal_list_command(mocker, requests_mock,
     assert get_mock.call_count == list_url_count
     assert next_link_get_mock.call_count == next_url_count
     assert ('?$top=' in str(get_mock.last_request)) == top_param_in_url
+
+
+@pytest.mark.parametrize('args, expected_args',
+                         [
+                             ({'id': 'TEST', 'app_id': None}, "/TEST"),
+                             ({'id': None, 'app_id': 'TEST'}, "(appId='TEST')"),
+                             ({'id': 'TEST', 'app_id': 'TEST'}, "/TEST")
+                         ])
+def test_remove_service_principals_command(mocker, requests_mock, args, expected_args):
+    """
+        Given:
+            - Required arguments (id or app_id or both)
+        When:
+            - Executing msgraph-apps-service-principal-remove command
+        Then:
+            - Ensure delete_service_principals client's method was called with the right argument
+    """
+    from MicrosoftGraphApplications import Client, remove_service_principals_command
+
+    mock_token = {'access_token': 'test_token', 'expires_in': '86400'}
+    requests_mock.post('https://login.microsoftonline.com/organizations/oauth2/v2.0/token', json=mock_token)
+    client = Client(app_id='TEST', verify=False, proxy=False, connection_type='TEST', tenant_id='TEST', enc_key='TEST')
```

Comment:
https://github.com/demisto/content/pull/29750/files#r1329960195

Reply:
Good to know, thanks. I changed it